Plugins.cpp

```diff
@@ -52,6 +58,10 @@ PluginInfo::PluginInfo() {
 }
 
 Plugins::Plugins(QObject *p) : QObject(p) {
+	// Enable all directories by default
+	bUseCurrentDirPlugins = true;
+	bUseSystemPlugins = true;
+	bUseUserPlugins = true;
```

Comment:
I would prefer member variables to be initialized with a [member initialization list](http://en.cppreference.com/w/cpp/language/initializer_list) - which is the equivalent to what we used before.

Reply:
Done.